lib/utils/timetable.ts

```diff
@@ -19,7 +20,22 @@ export const csvToObjectArray = <T>(CSVString: string): T[] => {
 export const objectArrayToCSVString = (
   objArr: { [key: string]: unknown }[]
 ): string => {
-  return unparse(objArr);
+  // ensure all objects keys are in the same order
+  const orderedObjectArray = objArr.map((object) =>
+    Object.keys(object)
+      .sort()
+      .reduce((obj: { [key: string]: unknown }, key) => {
+        obj[key] = object[key];
+        return obj;
+      }, {})
+  );
+
+  return unparse({
+    fields: Object.keys(orderedObjectArray[0]).map((header) =>
```

Comment:
This way of unparsing the objects allows us to specify names for the headers (so that we can use the kebab-case version of the object keys) but we have to make sure that all the objects in the array have the keys in the same order hence line 24